### PR TITLE
V2.13-RUN-10287

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/home/whats-new-2-13.md [hunk 1]
```diff
@@ -73,9 +73,14 @@ Added `suspend`/`stop` to [Submitting Workloads via HTTP/REST](../developer/clus
 
 <!-- RUN-9521/9522  Provide a description in CLI when command fails -->
 
+
+<!-- RUN-10287/10317/10313-10851 Show Node pools priority list according to workspace policy -->
+When there is a workspace policy in place, the node pool priority list in *Workspaces* is the same as the policy, and is editable when the workspace is built from scratch. The list will not be editable if the policy dictates that the list is locked. If a policy has not been enabled, then the priority list will come from either a template, or the project.
```
Comment (liohill) [inline docs/home/whats-new-2-13.md:78]: I think it is written in a confusing way, the message needs to be:
The selection of node pools is now incorporated in the workspace submission page, this allows data scientists to quickly determine the list of node pools to try by dragging & dropping to their desired order of priority.

Moreover, when the node pools priority list is locked by admin's policy this list will be freezed & the data scientist will be able to edit it even if the workspace is created from the template & duplicated

Comment (jasonnovichRunAI) [reply docs/home/whats-new-2-13.md]: See the new changes in the "Files Changed" link at the top.

File: docs/admin/workloads/policies.md [hunk 2]
```diff
@@ -20,9 +20,11 @@ Policies are specific to Workload type as such there are several kinds of Polici
 
 A Policy can be created per Run:ai Project (Kubernetes namespace). Additionally, a Policy resource can be created in the `runai` namespace. This special Policy will take effect when there is no project-specific Policy for the relevant workload kind.
 
+ When researchers create a new interactive workload or workspace, they see list of available node pools and their priority. Priority is set by dragging and dropping the node pools in the desired order of priority. When the node pool priority list is **locked** by an administrator **policy**, the node pool list isn't editable by the Researcher even if the workspace is created from a template or copied from another workspace.
```
Comment (liohill) [inline docs/admin/workloads/policies.md:23]:
```suggestion
 When researchers create a new workload or workspace, they see list of available node pools and their priority. Priority is set by dragging and dropping the node pools in the desired order of priority. When the node pool priority list is **locked** by an administrator **policy**, the node pool list isn't editable by the Researcher even if the workspace is created from a template or copied from another workspace.
```

File: docs/home/whats-new-2-13.md [hunk 3]
```diff
@@ -73,9 +73,14 @@ Added `suspend`/`stop` to [Submitting Workloads via HTTP/REST](../developer/clus
 
 <!-- RUN-9521/9522  Provide a description in CLI when command fails -->
 
+
+<!-- RUN-10287/10317/10313-10851 Show Node pools priority list according to workspace policy -->
+Added Node pool selection as part of the workspace submission form. This allows researchers to quickly determine the list of node pools available and their priority. Priority is set by dragging and dropping them in the desired order of priority. In addition, when the node pool priority list is locked by an administrator policy, the list isn't editable by the Researcher even if the workspace is created from a template or copied from another workspace.
```
Comment (liohill) [inline docs/home/whats-new-2-13.md:78]:
```suggestion
Added Node pool selection as part of the workload and workspace submission form. This allows researchers to quickly determine the list of node pools available and their priority. Priority is set by dragging and dropping them in the desired order of priority. In addition, when the node pool priority list is locked by an administrator policy, the list isn't editable by the Researcher even if the workspace is created from a template or copied from another workspace.
```

**Pull request merged**